repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/multi-org-logo.ftl

```diff
@@ -13,47 +13,48 @@
 
 <#if document?? && document.logoGroup?? && document.logoGroup.logos?size gt 0>
     <#--  Multi org logo: START  -->
-    <div class="nhsuk-header__multi-logo">
-        <#list document.logoGroup.logos as logo>
-            <#--  Builds link  -->
-            <#if logo.linkDocument??>
-                <#assign href=getInternalLinkURL(logo.linkDocument)>
```

Comment:
Any reason why this line has been replaced with `<@hst.link hippobean=logo.linkDocument/>` ? Because we need `<#assign href=getInternalLinkURL(logo.linkDocument)>` to render Mini-hub child Standard content page URLs.

Reply:
re-instated.